packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -9,6 +9,7 @@ import 'dart:developer';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
 
+import 'binding.dart' show WidgetsBinding;
```

Comment:
why the 'show'?

Reply:
binding.dart imports framework.dart, so I wanted to be clear about the limited scope of the import here.  But maybe it doesn't matter...

Reply:
I'd probably do it the other way around then (have the 'show' in binding). For consistency with the other libraries.

Reply:
I just removed it from here for consistency with other libraries (namely rendering library has this case, and it doesn't use `show` in either place).